packages/@nucleus/tests/dummy/app/templates/components/nucleus-button/demo-2.hbs

```diff
@@ -0,0 +1,10 @@
+{{#docs-demo as |demo|}}
+  {{#demo.example name="nucleus-button-group.hbs"}}
+  {{#nucleus-button-group  as |button|}}
+    {{#button.button type="toggle"}}Button1{{/button.button}}
```

Comment:
The entire button-group can either be toggle or plain buttons. Not the individual buttons inside them.